flask_resty/view.py

```diff
@@ -107,8 +108,62 @@ def validate_request_id(self, data, expected_id):
         if id != expected_id:
             raise ApiError(409, {'code': 'invalid_id.mismatch'})
 
+    def update_spec(self, path, spec, many=False, tag=False, **kwargs):
```

Comment:
Literally just pull it out into a `ApiSpecViewMixin` or something. Then if they want this behavior, they can do

```py
class WidgetView(GenericModelView, SpecViewMixin)
```